modules/ROOT/pages/enable-fips.adoc

```diff
@@ -14,6 +14,8 @@ The Federal Information Processing Standard (FIPS) 140-2 is a US government secu
 
 FIPS enablement is important for many users, particularly if you work for or with US government agencies. Running your Open Liberty servers on a FIPS-compliant JVM helps ensure that only FIPS-certified cryptography is used when an application uses Java security libraries or APIs. FIPS-compliant JVM options for Open Liberty are link:https://www.ibm.com/docs/en/sdk-java-technology/8[IBM SDK, Java Technology Edition] or link:https://developer.ibm.com/articles/explore-options-for-downloading-ibm-semeru-runtimes[IBM Semeru Runtimes].
 
+NOTE: To enable FIPS 140-3, you must use IBM SDK for Java 8.0.8.30 or later.
+

```

Comment:
combine this info with following paragraph. You might need to revise the text to make it clear that if you want to use 140-3, you must use the IBM SDK and follow the steps in the WL doc. I wouldn't mark it "NOTE"- we dont generally use notes in OL